woocommerce/woocommerce
# Add reorder grouped products option

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR disables the re-order functionality in the add products modal and instead adds a `Reorder` button to allow users to reorder the list within a specific modal.

https://github.com/woocommerce/woocommerce/assets/2240960/2ea2add7-2c94-4000-9695-64d9b435deaa

Closes #41965  .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure feature `New product editor` is enabled under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`
2. Make sure feature `product-grouped` is enabled under `Features` tab from `/wp-admin/tools.php?page=woocommerce-admin-test-helper` (WooCommerce Beta Tester plugin) is required
3. Go to **WooCommerce > Home** and click on the **Add products** task list ( the top one )
4. Click the Grouped product item
5. Add a name and price and scroll down to the `Products in this group` section
6. There should be no `Reorder` button visible
7. Click `Add products` button.
8. Search some product in the modal and add them ( NOTE: the drag handle should not be available in this modal )
9. Once you added products a `Reorder` button should be visible
10. Click the `Reorder` button and re order the items using the drag handle and click `Done`
11. Save the product and click `Preview` the products should be shown in order as previously set.
12. Try to `reorder` them again, the form should show as dirty ( once changed ).



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
